oc-chef-pedant/spec/api/user_spec.rb

```diff
@@ -64,6 +64,52 @@
             })
         end
 
+        context "saml user" do
+          let(:username) do
+            "user_with_external_uid"
+          end
+
+          let(:external_auth_id) do
+            "sam.l.jackson"
+          end
+
+          let(:user_options) do
+            { :overrides => {
+                :first_name => "external",
+                :last_name => "user",
+                :display_name => "SAML USER",
+                :external_authentication_uid => external_auth_id
+              }
+            }
+          end
+
+          let(:filtered_external_users_body) do
+            { username => "#{request_url}/#{username}" }
+          end
+
+          before :each do
+            @user = platform.create_user(username, user_options)
+          end
+
+          after :each do
+            platform.delete_user(@user)
+          end
+
+          it "returns no users when filtering by non-existing external_authentication_uid", :smoke do
+            get("#{request_url}?external_authentication_uid=somenonexistingemail@somewhere.com", platform.superuser).should look_like({
```

Comment:
Nitpick, looking at [chef-manage's test stubs](https://github.com/chef/chef-manage/blob/master/src/chef-manage/spec/controllers/sessions_controller_spec.rb#L49-L50), perhaps this should be `%40`... doesn't matter much for this test, though.